packages/lesswrong/server/recommendations.js

```diff
@@ -159,18 +160,78 @@ const getRecommendations = async ({count, algorithm, currentUser}) => {
   }
 };
 
+const getResumeSequences = async (currentUser, context) => {
+  if (!currentUser)
+    return [];
+  if (!currentUser.partiallyReadSequences)
+    return [];
+  
+  return Promise.all(_.map(currentUser.partiallyReadSequences,
+    async partiallyReadSequence => ({
+      sequence: partiallyReadSequence.sequenceId
+        ? await context["Sequences"].loader.load(partiallyReadSequence.sequenceId)
+        : null,
+      collection: partiallyReadSequence.collectionId
+        ? await context["Collections"].loader.load(partiallyReadSequence.collectionId)
```

Comment:
I would deconstruct these in the function signature instead of referencing them this way.